spec/asyncapi.md

```diff
@@ -8,7 +8,7 @@ This version is not yet ready to be used. We're currently working on it. If you
 
 Part of this content has been taken from the great work done by the folks at the [OpenAPI Initiative](https://openapis.org). Mainly because **it's a great work** and we want to keep as much compatibility as possible with the [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification).
 
-#### Version 2.5.0
+#### Version 3.0.0
```

Comment:
If I were you, I'd leave these changes (2.5.0 -> 3.0.0) out of this PR. If you keep them, you're gonna get conflicts because we already released 2.6.0. So yeah, my recommendation is that you go through all the document and undo this kind of change.

Reply:
fixed. When we are done, done... i will squasch all commits to avoid merge problems here.